qmxgraph/page/api.js

```diff
@@ -706,6 +702,48 @@ graphs.Api.prototype.resizeContainer = function resizeContainer (width, height)
     this._graphEditor.graph.doResizeContainer(width, height);
 };
 
+/**
+ * Zoom in the graph.
+ */
+graphs.Api.prototype.zoomIn = function zoomIn () {
+    "use strict";
+    this._graphEditor.graph.zoomIn();
```

Comment:
Does the underlying API support a parameter that controls how much zoom is applied, like a factor or something?

Reply:
There is a `zoomFactor`, but that is `graph` member (not an function argument).
I could receive am argument here, use the zoom factor and restore the original value after.
Is that good or you think I should also expose the `zoomFactor` member?

Reply:
Nevermind, I asked just in case the zoomFactor was an argument already.